components/Dialogs/src/Theming/MDCAlertController+MaterialTheming.m

```diff
@@ -23,6 +23,10 @@
 @implementation MDCAlertController (MaterialTheming)
 
 - (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
+  // TODO(https://github.com/material-components/material-components-ios/issues/6637 ): Force-load
+  // the view before setting the buttons fonts to ensure that they aren't overwritten in
+  // -updateButtonFont. If the bug gets fixed, this hack can be removed.
+  (void)self.view;
```

Comment:
loadViewIfNeeded is preferred on iOS 9 and above: https://developer.apple.com/documentation/uikit/uiviewcontroller/1621446-loadviewifneeded

Please call that method when available, otherwise fall back to `self.view`.